scriptler/checkNodesLauncherVersion.groovy

```diff
@@ -12,19 +12,32 @@ import hudson.remoting.Launcher
 import hudson.slaves.SlaveComputer
 import jenkins.model.Jenkins
 
-def expectedVersion = Launcher.VERSION
-println "Expected Version = ${expectedVersion}"
+def expectedAgentVersion = Launcher.VERSION
+def expectedJavaVersion = System.getProperty("java.version")
+println "Master"
+println " Expected Agent Version = '${expectedAgentVersion}'"
+println " Expected Java Version = '${expectedJavaVersion}'"
 Jenkins.instance.getComputers()
         .findAll { it instanceof SlaveComputer }
         .each { computer ->
+    println "Node '${computer.name}'"
```

Comment:
I actually intended to print the Node name and then use the indentation as following:

```
Node 'myNode'
 is disconnected
Node 'myNode2'
 expected agent version '1.2.3' but got '3.2.1'
 expected java version '1.8.0_131' but got '1.8.0_121'
Node 'myNode3'
 OK
```

Maybe I can use a single line if OK or disconnect and several line if there are version mismatches.